src/services/LokiService.jsx

```diff
@@ -107,13 +106,7 @@ class LokiService {
   };
 
   saveDB = () => {
-    this.db.saveDatabase(function (err) {
-      if (err) {
-        console.log(`error : ${err}`);
-      } else {
-        console.log('database saved.');
-      }
-    });
+    this.db.saveDatabase(() => {});
```

Comment:
This logging tells us if we have an error with a db save. It should stay

Reply:
Okay, I'll leave the console logging in for now. Once we get more structured logging in we can revisit these.